packages/core/lib/commands/debug/meta.js

```diff
@@ -70,6 +77,6 @@ module.exports = {
           "Forces the debugger to use artifacts even if it detects a problem.  Dangerous; may cause errors."
       }
     ],
-    allowedGlobalOptions: ["network", "config"]
+    allowedGlobalOptions: ["config"] //network is allowed but is handled separately above so usage can be combined w/url
```

Comment:
It might be useful to document this is a workaround (or needed?) to handle a mutually exclusive inherited option?

Reply:
This comment is intended to compactly express that! :P  Do you think it's worth writing more here?

Reply:
I think it's worth the extra context to document the motivation because this change introduces a new technique to handle the case of displaying a mutually exclusive global and (local?) option.

Reply:
Are you saying to explicitly state that this is done to document the fact that "url" and "network" are mutually exclusive?

Reply:
yes @eggplantzzz

Reply:
OK, I've updated the comment to be more explicit.  I'll merge this once it passes CI.